corehq/apps/reports/templates/reports/partials/save_reports_modal.html

```diff
@@ -48,6 +48,7 @@ <h3 data-bind="text: modalTitle"></h3>
                                 <option value="last7">{% trans "Last 7 days" %}</option>
                                 <option value="last30">{% trans "Last 30 days" %}</option>
                                 <option value="lastmonth">{% trans "Last month" %}</option>
+                                <option value="lastyear">{% trans "Last year" %}</option>
```

Comment:
I think if that's very easy it's worth doing. at a minimum the daily form stats should not expose this since it will [pull one row out of the DB for every form submitted in the domain in the period](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/standard/monitoring.py#L539).